keanu-python/keanu/context.py

```diff
@@ -31,6 +35,14 @@ def __init__(self):
 
         self.__jvm_view = self._gateway.new_jvm_view()
 
+    def __build_classpath(self):
+        keanu_path = os.path.join(PATH, "classpath", "*")
+        nd4j_path = os.environ.get(ND4J_CLASSPATH_ENVIRONMENT_VARIABLE)
+        if nd4j_path is None:
+            return keanu_path
+        else:
+            return os.pathsep.join([keanu_path, os.path.join(nd4j_path, "*")])
```

Comment:
Is this for appveyor?

Reply:
yes, it's for Windows.

For now, Windows users have to download a 40MB jar (mkl-dnn) and set an environment variable to point to it

Reply:
😢